tests/python/pants_test/backend/python/tasks/test_interpreter_selection_integration.py

```diff
@@ -30,6 +30,40 @@ def hermetic(cls):
     # in the below tests.
     return True
 
+  def _build_pex(self, binary_target, config=None, args=None, version=PY_27):
+    # By default, Avoid some known-to-choke-on interpreters.
+    constraint = '["CPython>=3.6,<4"]' if version == PY_3 else '["CPython>=2.7,<3"]'
```

Comment:
This is the only change made to these 3 files. Using a ternary expression to better express intent and have less lines.